example_project/chilled_water_storage_scenario.csv

```diff
@@ -0,0 +1,14 @@
+Feature Id,Feature Name,Mapper Class
+1,Mixed_use 1,URBANopt::Scenario::ChilledWaterStorageMapper
+2,Restaurant 1,URBANopt::Scenario::ChilledWaterStorageMapper
+3,Restaurant 10,URBANopt::Scenario::ChilledWaterStorageMapper
+4,Restaurant 12,URBANopt::Scenario::ChilledWaterStorageMapper
+5,Restaurant 14,URBANopt::Scenario::ChilledWaterStorageMapper
+6,Restaurant 15,URBANopt::Scenario::ChilledWaterStorageMapper
+7,Office 1,URBANopt::Scenario::ChilledWaterStorageMapper
+8,Hospital 1,URBANopt::Scenario::ChilledWaterStorageMapper
+9,Hospital 2,URBANopt::Scenario::ChilledWaterStorageMapper
+10,Mixed use 2,URBANopt::Scenario::ChilledWaterStorageMapper
+11,Restaurant 13,URBANopt::Scenario::ChilledWaterStorageMapper
+12,Mall 1,URBANopt::Scenario::ChilledWaterStorageMapper
+13,Hotel 1,URBANopt::Scenario::ChilledWaterStorageMapper
```

Comment:
Could you add the additional (residential) features to this file? I think you are using the example_project_combined.json file by default.  and a blank line at the end